src/scenes/App/index.js

```diff
@@ -24,10 +25,7 @@ const { toggleAll } = appActions;
 
 export class App extends React.PureComponent {
   render() {
-    const {
-      classes,
-      txReturn,
-    } = this.props;
+    const { classes, langHandler, locale, txReturn } = this.props;
```

Comment:
can remove `locale` since we're grabbing it in `TermsAndConditions`

Reply:
yup forgot to remove done